bylaws.md

```diff
@@ -1,58 +1,111 @@
-2. Provisions of these Bylaws shall be subordinate to those of the Constitution. 
+## Article I: Definitions
 
-## Article II: Election of the Officers
+1. A member will be considered an **Active Member** if they have attended at least 50% of general body meetings per semester when they have not been considered “away.” A member is considered “away” if he or she is participating in an off-campus MQP, IQP, Co-op, or Humanities Project. Regular members who have not attended at least one meeting shall not be considered active.
+2. A **Quorum** shall consist of two-thirds of the Active Members of the Chapter in any regular or special meeting.
+3. A **Vote** shall consist of votes in person or, at the discretion of the Executive Board, votes by proxy but such proxy votes shall not count towards quorum.
 
-1. The Officers of the Chapter shall be elected at the first business meeting after the election of new Members during the last term of the year (D Term).
-2. Installation of Officers shall take place at the meeting during which they are elected.
+## Article II: Adoption and Amendment
+
+1. These bylaws shall become effective immediately upon adoption.
+2. These bylaws shall remain in effect until revised or amended.
+3. Any provision of these bylaws may be temporarily suspended by a two-thirds affirmative vote of the Active Membership of the chapter. The period of which the suspension is valid shall be determined at the time of the vote.
+4. These bylaws may be suspended in whole or in part by a unanimous vote of the Executive Board with the consent of the Chapter Advisor.
+5. Any proposed amendment to these bylaws must be distributed to all members no later than two weeks prior to any vote on the adoption of such amendment.
+6. The adoption of any amendment to these Bylaws shall require a two-thirds affirmative vote of the Active Membership of the Chapter at a regular or special meeting.
+
+## Article III: Membership
+
+1. This Chapter shall consist of Members and Honorary Members.
+2. The membership of the Chapter shall be made up of individuals whose academic achievements, reputations and creative abilities deserve recognition, and whose membership would enhance the stature of the Association. Specifically, the membership shall include graduate students, undergraduate students, former students, and faculty in the Computer Science Department at Worcester Polytechnic Institute, elected according to the procedures detailed in Article VI of these Bylaws.
+3. Honorary Members shall be individuals of distinguished achievement in Computer Science.
+4. Eligibility for membership shall not be restricted according to race, nationality, creed, color, sexual orientation or sex.
+
+## Article IV: Chapter Officers
+
+### Section I: Composition of the Executive Board
+
+1. The society shall elect three primary executive officers: a President, Secretary, and Treasurer.
```

Comment:
I thought we needed a VP for the sake of being considered a club? Or am I remembering that incorrectly? In the event that I am in fact remembering that correctly, does provision 3 handle that case?